exercise/views.py

```diff
@@ -228,6 +239,52 @@ def submission_check(self, error=False, request=None):
         )
         return submission_status, submission_allowed, issues, students
 
+    def get_page(self, request: HttpRequest, students: List[UserProfile]) -> ExercisePage:
+        """
+        Determines which page should be displayed for this exercise:
+        1) If the `draft` URL parameter is not `false`, and there is an active
```

Comment:
This part is outdated, should be "If the `draft` URL parameter is `true`". I'll fix this soon.

Reply:
I fixed this and force-pushed.